platform/fabric/core/provider.go

```diff
@@ -149,13 +149,14 @@ func (p *fnsProvider) newFNS(network string) (driver.FabricNetworkService, error
 	}
 
 	// Local MSP Manager
+	cacheSize := 500
```

Comment:
is this some default cache size? can this value still change via config - is this happening in `msp/service.go` Line 405-408?

If so, I would rename this variable here to defaultCacheSize with a comment that this value may be overritten by using a config file

Reply:
very good, let me do that.